microsoft/vscode
# Add `workbench.quickInput.ignoreBuiltInKeybindings`

# Summary

This PR adds the `workbench.quickInput.ignoreBuiltInKeybindings` setting to solve the following issues:

- #98479
- #193839

When the setting is set to `true`, all hardcoded quick input keybindings are ignored.

# Manual Testing

> I'm not familiar with the automated tests, but if there is a good way to add tests for these cases, point me to some docs/similar tests and I can give it a try

- Add following to keyboard bindings:
```
{
    "command": "workbench.action.focusPanel",
    "key": "pagedown" // Also try with up, down, left, right, pageup, home, end
},
```
- Open any quick input while in editor focus
- Press `pagedown` key
- Verify panel was opened (prior to this PR, the `pagedown` key would instead just jump to last selection in list)

cc: @TylerLeonhardt
